internal/api/handler/get_public_config_v1alpha1.go

```diff
@@ -14,10 +15,21 @@ type GetPublicConfigV1Alpha1Func func(
 ) (*connect.Response[svcv1alpha1.GetPublicConfigResponse], error)
 
 func GetPublicConfigV1Alpha1(
-	cfg *svcv1alpha1.GetPublicConfigResponse,
+	cfg config.ServerConfig,
 ) GetPublicConfigV1Alpha1Func {
 	return func(context.Context, *connect.Request[svcv1alpha1.GetPublicConfigRequest],
 	) (*connect.Response[svcv1alpha1.GetPublicConfigResponse], error) {
-		return connect.NewResponse(cfg), nil
+		var oidcCfg *svcv1alpha1.OIDCConfig
+		if cfg.OIDCConfig != nil {
+			oidcCfg = &svcv1alpha1.OIDCConfig{
+				IssuerUrl: cfg.OIDCConfig.IssuerURL,
+				ClientId:  cfg.OIDCConfig.ClientID,
+				Scopes:    cfg.OIDCConfig.Scopes,
+			}
+		}
+		return connect.NewResponse(&svcv1alpha1.GetPublicConfigResponse{
+			AdminAccountEnabled: cfg.AdminConfig != nil,
+			OidcConfig:          oidcCfg,
+		}), nil
```

Comment:
Besides being unrelated to the version endpoint, I am wondering why we are reconstructing the response message every time now.